src/controllers/auth.js

```diff
@@ -48,7 +48,8 @@ export const verifyUser = async (req, res) => {
         res.write('file not found');
         res.writeHead(404);
       } else {
-        res.write(data);
+        const modifiedHtml = data.replace('%VERIFY_URL_PLACEHOLDER%', process.env.REDIRECT_URL);
```

Comment:
We already have an env variable named `FRONTEND_DOMAIN` which corresponds to REDIRECT_URL. We could use it here. We could also combine these 2 into one line, not much use of another variable here.